Tools/CISupport/ews-build/steps.py

```diff
@@ -312,7 +312,7 @@ def add_pr_details(self):
         owners = self.getProperty('owners', [])
         revision = self.getProperty('github.head.sha')
 
-        self.setProperty('change_id', revision, 'ConfigureBuild')
+        self.setProperty('change_id', revision[:HASH_LENGTH_TO_DISPLAY], 'ConfigureBuild')
```

Comment:
Does this affect other places we are using revision? Please verify affect on those places.

Reply:
If you look at the line above, this is only ever going to be a hash. It will also effect our build url, but that's sort of the point, our build download URL should also get this.

The builds I just linked are full a full build + test.

Reply:
I meant `change_id`.
I just checked and it affects UploadBuiltProduct, TransferToS3 and DownloadBuiltProduct. In those steps the url is constructed only based on this hash. So chances of collision are high. Although we store the builds only for 14 days, so the time-duration reduces the chances of collision. I'll let you decide if you actually want to use 8 characters git hash in those steps.

Reply:
We should be fine, we get about a 1% change of collision with 10,000 commits at 8 characters, and we have ~600 commits every 2 weeks. Figuring 10 iterations per commit, thats 6,000 hashes every 2 weeks, which is .5% chance of collision. Back-of-the-napkin calculation of .5% chance every 2 weeks means about a single collision every 5 years.